config/ufs/machines/config_batch.xml

```diff
@@ -262,4 +245,19 @@
     </queues>
   </batch_system>
 
+  <batch_system MACH="gaea" type="slurm" >
+    <batch_submit>sbatch</batch_submit>
+    <jobid_pattern>Submitted batch job (\d+) on cluster .*</jobid_pattern>
+    <submit_args>
+      <arg flag="--time" name="$JOB_WALLCLOCK_TIME"/>
+      <arg flag="--qos" name="$JOB_QUEUE"/>
+      <arg flag="--account" name="$CHARGE_ACCOUNT"/>
+    </submit_args>
+    <directives>
+      <directive>--clusters=c4</directive>
+    </directives>
+    <queues>
+      <queue walltimemax="00:30:00" nodemin="1" nodemax="171">debug</queue>
```

Comment:
This is good for testing but is there another queue you should add for production use?  there is a default attribute that you can use to set the preferred queue if they overlap

Reply:
Yes, I think when our testers work with this the next days on the larger test cases, they will need to add other queues (maybe - debug is fairly generous on gaea). I wanted to get the initial config going with the C96 case.